packages/theme/actionButton.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native'
+
+import { large } from 'theme/include/fonts'
+import { white } from 'theme/include/colors'
+import { mediumHeight } from 'theme/include/icon'
+
+export default StyleSheet.create({
+  actionButtonIcon: {
```

Comment:
Already within the context of `actionButton` - this should be `actionButton.icon` and not the current `actionButton.actionButtonIcon`.